pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/helm-managed.test.yaml

```diff
@@ -27,3 +27,14 @@ tests:
     envVars(.deployments.sensor; "sensor") | assertThat(.["ROX_HELM_CLUSTER_CONFIG_FP"] == "test-fpr")
   set:
     meta.configFingerprintOverride: "test-fpr"
+
+- name: "Operator-installed instance should"
+  set:
+    managedBy: MANAGER_TYPE_KUBERNETES_OPERATOR
+  tests:
+  - name: "be not Helm-managed"
+    expect: |
+      helmClusterConfig | assertThat(.notHelmManaged == true)
```

Comment:
Again, this seems wrong. We **don't** want the upgrader to touch operator-managed clusters, right?

Reply:
Oh, I missed this one. Moriz used that to fix the `notHelmManaged` field, whereas I later removed it completely. 
I will replace this test with something that is up to date.

Reply:
Done. See [d818103](https://github.com/stackrox/stackrox/pull/12935/commits/d818103e3c76ec43b5ca1ae88eb9e5ea2309fb77)